tests/foreman/ui/test_branding.py

```diff
@@ -0,0 +1,57 @@
+"""Tests for branding
+
+:Requirement: Branding
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Branding
+
+:Assignee: akjha
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo.config import settings
+
+
+def test_check_version_login_screen(default_sat):
+    """Check correct version on the login screen
+
+    :id: f48110ad-29b4-49b1-972a-a70075a05732
+
+    :Steps: Curl the login screen
+
+    :expectedresults: Correct version depending on the snap
+
+    :CaseLevel: System
+
+    :BZ: 1315849, 1367495, 1372436, 1502098, 1540710, 1582476, 1724738, 1959135, 2076979
+    """
+    result = default_sat.execute(
+        f'curl -k {default_sat.url}/users/login | '
+        f'grep -e "LoginPage" -e "{settings.server.version.release}"'
+    )
```

Comment:
It would be nice if you could add some checks here for satellite/satellite-capsule RPMs as well or are you planning to add additional tests for it?

Reply:
Could you help me understand `checks here for satellite/satellite-capsule RPMs`? I wonder what you are saying falls under branding component.

Reply:
Why curl this page and not use widgetastic/browser integration to make these assertions?  There should be an element in the view that contains the release, and you should be able to read that widget.

Reply:
Yep, did that